common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/OAuth2TokenCache.java

```diff
@@ -54,4 +53,12 @@ public OAuth2TokenCache(Context context) {
     public abstract void saveTokens(final T oAuth2Strategy,
                                     final U request,
                                     final V response) throws ClientException;
+
+    protected Context getContext() {
```

Comment:
This should be removed. Doesn't make sense to expose this as API

Reply:
mContext has be private and needs it own getter and setter based on Checkstyle's warning. mContext gets called in ADALOAuth2TokenCache. This is reason I add this getter as protected method here.

Reply:
mContext has be private and needs it own getter and setter based on Checkstyle's warning. mContext gets called in ADALOAuth2TokenCache. This is reason I add this getter as protected method here.

`[ant:checkstyle] [WARN] ..\common\src\main\java\com\microsoft\identity\common\internal\providers\oauth2\OAuth2TokenCache.java:34:29: Variable 'mContext' must be private and have accessor methods. [VisibilityModifier]`

Reply:
👍